apps/cowswap-frontend/src/modules/limitOrders/containers/LimitOrdersWarnings/index.tsx

```diff
@@ -75,12 +75,7 @@ export function LimitOrdersWarnings(props: LimitOrdersWarningsProps) {
 
   const canTrade = localFormValidation === null && primaryFormValidation === null && !tradeQuote.error
   const showPriceImpactWarning =
-    canTrade &&
-    !tradeQuote.isLoading &&
```

Comment:
The warning on Limit orders page was depending on quote loading state, this is not correct